pac4j-core/src/main/java/org/pac4j/core/engine/DefaultCallbackLogic.java

```diff
@@ -47,6 +47,12 @@ public R perform(final C context, final Config config, final HttpActionAdapter<R
         } else {
             defaultUrl = inputDefaultUrl;
         }
+        final boolean saveInSession;
+        if (inputSaveInSession == null) {
```

Comment:
I'm a bit skeptical about the use case which justifies not to save the user profile into the web session after a remote login process: can you refresh my memory about your use case?

Reply:
Well the use case is the following, a spa relying on a jaxws backend request authent on a saml2 idp.
Upon auth completion the idp will trigger callback filter by the mean of the browser.
The callbackfilter will then produce a samlprofile and place it in the request context. 
This profile will be changed into a jwt token and sent back to the spa client.
In this use case as the final callback is processed in one single call there is no need for a session creation. 
The spa will authenticate itself using the jwt token for further requests.

Reply:
OK, it's a bit unusual, but it makes sense.